mmv1/third_party/tgc_next/tfplan2cai/converters/services/resourcemanager/project.go

```diff
@@ -0,0 +1,149 @@
+package resourcemanager
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/terraform-google-conversion/v6/pkg/caiasset"
+	"github.com/GoogleCloudPlatform/terraform-google-conversion/v6/pkg/tfplan2cai/converters/cai"
+
+	"github.com/hashicorp/terraform-provider-google-beta/google-beta/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google-beta/google-beta/transport"
+
+	"google.golang.org/api/cloudbilling/v1"
+	"google.golang.org/api/cloudresourcemanager/v1"
+)
+
+func ResourceConverterProject() cai.ResourceConverter {
+	return cai.ResourceConverter{
+		Convert: GetProjectAndBillingInfoCaiObjects,
+	}
+}
+
+func GetProjectAndBillingInfoCaiObjects(d tpgresource.TerraformResourceData, config *transport_tpg.Config) ([]caiasset.Asset, error) {
+	if projectAsset, err := GetProjectCaiObject(d, config); err == nil {
+		assets := []caiasset.Asset{projectAsset}
+		if billingAsset, err := GetProjectBillingInfoCaiObject(d, config); err == nil {
+			assets = append(assets, billingAsset)
+			return assets, nil
+		} else {
+			return []caiasset.Asset{}, err
+		}
+	} else {
+		return []caiasset.Asset{}, err
+	}
+}
+
+func GetProjectCaiObject(d tpgresource.TerraformResourceData, config *transport_tpg.Config) (caiasset.Asset, error) {
+	linkTmpl := "//cloudresourcemanager.googleapis.com/projects/{{number}}"
+	name, err := cai.AssetName(d, config, linkTmpl)
+	if err != nil {
+		return caiasset.Asset{}, err
+	}
+	if data, err := GetProjectData(d, config); err == nil {
+		return caiasset.Asset{
+			Name: name,
+			Type: "cloudresourcemanager.googleapis.com/Project",
+			Resource: &caiasset.AssetResource{
+				Version:              "v1",
+				DiscoveryDocumentURI: "https://cloudresourcemanager.googleapis.com/$discovery/rest?version=v1",
```

Comment:
Is this URL used for test/verification purpose?

Reply:
No. This url is from the real CAI assets returned from the CAI APIs, for example, https://cloud.google.com/asset-inventory/docs/reference/rest/v1/assets/list